articles/libraries/overview.md

```diff
@@ -36,7 +36,13 @@ The Lock widget is a simple way to integrate Auth0 into existing projects and pr
 
 ## Auth0 SDKs
 
-These SDKs for Auth0 allow you to trigger the authentication process, parse JWTs, and authenticate requests to your APIs. There is no UI or login widget provided alongside the SDKs; they are for implementation alongside a custom UI. For a complete listing of supported SDKs, along with links to their GitHub repositories, take a look at the Auth0 [Support Matrix](/support/matrix#sdks). For more detailed documentation on our most popular SDKs, take a look below:
+These SDKs for Auth0 allow you to trigger the authentication process, parse JWTs, and authenticate requests to your APIs. There is no UI or login widget provided alongside the SDKs; they are for implementation alongside a custom UI. 
```

Comment:
You can improve on that by simplifying more the wording and adding some links along the way. For example:

> These SDKs enable you to log in users, parse [JWTs](/jwt), and [access your APIs](/api-auth). 
> No UI is included, they are meant to be used with your own custom UI.